client/consensus/grandpa/interfaces.go

```diff
@@ -42,3 +42,19 @@ type AuxStore interface {
 	// Get Query auxiliary data from key-Value store.
 	Get(key []byte) (*[]byte, error)
 }
+
+type HashI interface {
+	constraints.Ordered
```

Comment:
why do we need to hashes be ordered? since they are hashes how they can be ordered?

Reply:
Its because in the finalityGrandpaPackage hashes are stored in the btree which requires an ordering property. It doesn't need to be ordered for our own purposes, but it is needed for btree storage